File: 1159.pas
program URI1159;

{$APPTYPE CONSOLE}


uses
  SysUtils, Math;
  
function isPar(var x: int64): boolean;
begin
	isPar := (x mod 2) = 0;
end;
var
	i : integer;
	val, x, soma : int64;
begin
	repeat
		soma := 0;
		readln(val);
		if val = 0 then continue;
		x := val;
		if not(isPar(x)) then x := x + 1;
		for i := 0 to 4 do
		begin
			soma := soma + (x + (i * 2));
		end;
		writeln(soma);
	until val = 0;
end.

File: 1159.c
#include <stdio.h>
int main() {
	int x, i, val, soma;
	do {
		soma = 0;
		scanf("%d", &val);
		if (val == 0) continue;
		if (val % 2 != 0) x = val + 1;
		else x = val;
		for (i = 0; i < 5; i++) {
			soma += (x + (i * 2));
		}
		printf("%d\n", soma);
	} while(val != 0);
}
